Production_scripts/Process_extracts/extract_delayed_discharges.R

```diff
@@ -0,0 +1,110 @@
+#####################################################
+# Draft pre processing code for Delayed Discharges
+# Author: Jennifer Thom
+# Date: April 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input - Jul16_{latest dd period}DD_LinkageFile.zssav
+# Description - Preprocessing of raw delayed discharges file.
+#               Tidy up file in line with SLF format
+#                prior to processing.
+#####################################################
+
+# Load Packages #
+library(dplyr)
+library(janitor)
+library(lubridate)
+library(createslf)
+
+
+# Read in data---------------------------------------
+
+year <- check_year_format("1920")
+
+dd_file <- haven::read_sav(get_dd_path(ext = "zsav")) %>%
```

Comment:
A trivial question, and sorry if I made a mistake here. Just wondering if we are supposed to rds files rather than zsav ones.

Reply:
We currently get the file in SPSS format (zsav) from the DD team, as they've not yet translated their code. We'll get (hopefully) the same file in rds format in the future.